content/eu/about/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Honi buruz
+description: Zer da Moodleko Tresnen Gida?
+---
+
+_Moodleko Tresnen Gida_ [Moodle][moodle] 3 edo bertsio berriago bat erabiltzeko irakasle, hezitzaile eta trebatzaileentzat diseinatutako erreferentziazko taula da. Moodleren tresna nagusiei buruzko informazioa ematen du, eta lortu nahi den helburu pedagogikoaren arabera tresnarik egokienak aukeratzen laguntzen du. Tresna bakoitza erabiltzean lor daitezkeen [Bloom-en maila taxonomikoak][bloom] ere aipatzen ditu.
+
+Tresna hau [Eastern Institute of Technology](https://www.eit.ac.nz/)-ko (Zeelanda Berria) [Joyce Seitzinger](https://twitter.com/catspyjamasnz)-en ideia izan zen, 2020ko maiatzean Moodle 3.x-erako [Nicolas Martignoni][nm]-eko egokitua [Gavin Henrick](https://twitter.com/ghenrick)-ek 2 bertsiorako egindako lanean oinarrituta.
```

Comment:
This line should be a translation of "An idea by Joyce Seitzinger of the Eastern Institute of Technology (NZ), updated in May 2020 for Moodle 3.x by Nicolas Martignoni and translated into Basque by Iñigo Zendegi."

Reply:
I've updated that text both here and in config.yml